content/docs/tutorials/cloud-tutorial.md

```diff
@@ -0,0 +1,305 @@
+
+%copy first-line%
+```bash
+$ fluvio cloud login
+InfinyOn Cloud email: John@example.com
+Password:
+```
+
+Use the `fluvio profile list` command to confirm that your CLI is linked to the
+InfinyOn Cloud instance.
+
+%copy first-line%
+```bash
+$ fluvio profile list
+     PROFILE   CLUSTER   ADDRESS                     TLS
+  *  cloud     cloud     router.infinyon.cloud:9003  Verified
+```
+
+## A Quick Introduction to the Fluvio Interface
+
+You can now see Fluvio in action by creating a simple topic, and pushing data to it:
+
+%copy first-line%
+```bash
+$ fluvio topic create greetings
+topic "greetings" created
+```
+
+%copy first-line%
+```bash
+$ echo 'Hello world!' | fluvio produce greetings
+
+```
+
+The data you store is viewable from both the command line through Fluvio CLI and online
+through the Cloud interface.
+
+### Fluvio CLI
+
+The Fluvio CLI is currently the easiest way to interact with the Fluvio database.
+
+Two core commands of Fluvio you will want to be familiar with are `fluvio produce <topic>` and `fluvio consume <topic>`
+
+#### Produce
+
+`fluvio produce` is the main way to get data into the Fluvio database. While most of the time
+you may be calling it through an API, here is how to access it through the CLI.
+
+%copy first-line%
+```bash
+$ fluvio produce greetings
+> test
+Ok!
+> meow
+Ok!
+>
+```
+
+-> To quit the continuous mode of `produce` press `CTRL-C`.
+
+`fluvio produce <topic> [flags]` takes in either input from stdin, or from a file. The
+stdin input can be piped into [like in this tutorial](#an-easy-fluvio-script--bash),
+or filled continuously as seen just above. When running in continuous mode, it won't stop
+until it is killed by the user.
+
+Some useful option flags to be aware of:
+
+* `-f <file name>` – to use a file as input to be read and uploaded as multiple records.
+* `--raw` – to specify that the incoming data should be stored as a single record.
+
+#### Consume
+
+`fluvio consume` is the main way to read out data from the Fluvio database. Either
+in scripting, or through the use of the CLI, most actions will use `consume` in some way.
+
+Here is an example of Consume in action:
+
+%copy first-line%
+```bash
+$ fluvio consume greetings -dB
+Consuming records from the beginning of topic 'greetings'
+Hello world!
+test
+meow
+```
+
+-> To quit the continuous mode of `consume` press `CTRL-C`.
+
+`fluvio consume <topic> [flags]` by default prints to the terminal new records as
+they appear. By default it runs nonstop until quit, the examples used here all
+use the `-d` flag to tell it to stop.
+
+Some useful option flags to be aware of:
+
+* `d` – to halt consumption after reaching the end of the records available.
+* `T[int]` – to specify that it should consume only the T(default 10) most recent records.
+* `B[int]` – to specify to start consuming records B(default 0) after the start of the database.
+
+
+### InfinyOn Cloud Interface
```

Comment:
Let's have a conversation about this. We should stay away from sending users to Cloud.

Once we send them to the CLI, we need to keep them there.
Let's have a conversation about what we should do next.

Reply:
I'll stop reviewing now until we discuss.